server/block/air.go

```diff
@@ -12,6 +20,17 @@ func (Air) HasLiquidDrops() bool {
 	return false
 }
 
+// Ignite ...
+func (Air) Ignite(pos cube.Pos, w *world.World) bool {
```

Comment:
I really don't like the idea of Air being "ignitable", I feel it should just be a default case in the flint and steel implementation.

Reply:
Changed.